pkg/sql/compile/remoterun_test.go

```diff
@@ -305,7 +305,7 @@ func Test_convertToVmInstruction(t *testing.T) {
 		{Op: int32(vm.Order), OrderBy: []*plan.OrderBySpec{}},
 		{Op: int32(vm.Product), Product: &pipeline.Product{}},
 		{Op: int32(vm.ProductL2), ProductL2: &pipeline.ProductL2{}},
-		{Op: int32(vm.Projection), ProjectList: []*plan.Expr{}},
+		{Op: int32(vm.Projection), ProjectList: []*plan2.Expr{}},
```

Comment:
Are you sure about this change?   Why say, the next operator, Filter, takes plan.Expr?    Something wrong.